cypress/support/utils/taxrate.js

```diff
@@ -79,3 +92,17 @@ function applyTaxrate(taxrate) {
     cy.selectInListField('To_Country_ID', taxrate.countryTo, false);
   });
 }
+
+function applyActive(taxrate) {
+  describe(`Create new Taxrate ${taxrate.name}`, function() {
+    //cy.log(`Taxrate - activate - START (c_taxrate_id=${this.c_taxrate_id})`);
+    //cy.visitWindow('137', `${this.c_taxrate_id}`);
```

Comment:
i guess the cy.log might be useful to be commented back in for debugging, but i don't see the use of the `cy.visitWindow('137', `${this.c_taxrate_id}`);`` please add a comment on its purpose or remove